internal/controllers/machinedeployment/machinedeployment_controller.go

```diff
@@ -235,13 +235,15 @@ func (r *Reconciler) reconcile(ctx context.Context, cluster *clusterv1.Cluster,
 	}
 
 	if d.Spec.Strategy.Type == clusterv1.RollingUpdateMachineDeploymentStrategyType {
+		log.Info(fmt.Sprintf("MachineDeployment %s strategy type is set to: %s", d.Name, d.Spec.Strategy.Type))
```

Comment:
I'm not sure if we need this log statement as it just logs the current content of the strategy type field (same in l.246)

I think the most interesting logs for "MD creating/deleting MS" can be added in machinedeployment_rolling.go and machinedeployment_rollout_ondelete.go

Reply:
sure, I will rework the PR accordingly